src/collections/_documentation/platforms/python/gcp_functions.md

```diff
@@ -0,0 +1,54 @@
+---
+title: GCP Functions
+sidebar_order: 8
+---
+
+Install our Sentry SDK in the `requirement.txt` section:
+```python
+sentry_sdk
+```
+
+You can use the GCP Functions integration for the Python SDK like this:
+```python
+import sentry_sdk
+from sentry_sdk.integrations.serverless import serverless_function
+
+sentry_sdk.init(dsn="___PUBLIC_DSN___")
+
+@serverless_function
+def my_function(...): ...
+```
+
+Use the generic integration by calling the `serverless_function` decorator. Decorators wrap a function and modify its behavior. Then, deploy and test the function. Checkout Sentry's [gcp sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/python) for detailed examples.
+
+{% capture __alert_content -%}
+
+If you are using another web framework inside of AWS Lambda, the framework may catch exceptions before they are sent to Sentry. To avoid this, enable the framework-specific integration as well, if one exists. See [*Integrations*](/platforms/python/#integrations) for more information
```

Comment:
I think this alert about using a different, more specific integration is not relevant in this context.

Reply:
Taken